stream-chat-android-client-sample/src/main/java/io/getstream/chat/android/client/sample/DocumentationSamples.kt

```diff
@@ -214,7 +215,7 @@ fun queryUsers2() {
     val filter = Filters.`in`("id", listOf("jessica"))
     val offset = 0
     val limit = 10
-    val sort = QuerySort().desc("last_active")
+    val sort = QuerySort.desc(User::lastActive)
```

Comment:
So the API allows you to sort on custom fields and the field names are imploded. So **extraData.something** is specified as **something**.

Something to fix in a future release/when people ask for it.